pkg/scene/builder/builder_test.go

```diff
@@ -380,8 +380,8 @@ func TestSceneBuilder(t *testing.T) {
 
 	sceneWidgetID1 := id.NewWidgetID()
 	sceneWidgetID2 := id.NewWidgetID()
-	sceneWidget1 := scene.MustNewWidget(&sceneWidgetID1, pluginID, pluginExtension1ID, scenePropertyID, false)
-	sceneWidget2 := scene.MustNewWidget(&sceneWidgetID2, pluginID, pluginExtension2ID, scenePropertyID, true)
+	sceneWidget1 := scene.MustNewWidget(&sceneWidgetID1, pluginID, pluginExtension1ID, scenePropertyID, false, nil)
+	sceneWidget2 := scene.MustNewWidget(&sceneWidgetID2, pluginID, pluginExtension2ID, scenePropertyID, true, nil)
```

Comment:
You should test the build of WidgetAlignSystem. In particular, testing here is an essential part of maintaining the quality of the scene's build functionality so that it does not rely on manual test.

Reply:
This part is to do with a new Widget, so the nil is to do with the WidgetLayout. I will add that though!